### PR TITLE
More introduction in VPA readme

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @tkulczynski @kgrygiel

### CODE REVIEW

File: vertical-pod-autoscaler/README.md [hunk 1]
```diff
@@ -1,14 +1,46 @@
...
+### Intro
 
 Vertical Pod Autoscaler (VPA) frees the users from necessity of setting
 up-to-date resource requests for their containers in pods.
 When configured, it will set the requests automatically based on usage and
 thus allow proper scheduling onto nodes so that appropriate resource amount is
 available for each pod.
 
-# For users
+### Architecture
```
Comment (tkulczynski) [inline vertical-pod-autoscaler/README.md]: There's already a little section called Architecture below, could you please merge them?

File: vertical-pod-autoscaler/README.md [hunk 2]
```diff
@@ -1,14 +1,46 @@
...
-# Intro
+### Intro
 
 Vertical Pod Autoscaler (VPA) frees the users from necessity of setting
 up-to-date resource requests for their containers in pods.
 When configured, it will set the requests automatically based on usage and
 thus allow proper scheduling onto nodes so that appropriate resource amount is
 available for each pod.
 
-# For users
```
Comment (tkulczynski) [inline vertical-pod-autoscaler/README.md]: My view was that the README is split into two big sections "For users" and "For developers" which have relevant subsections. In my opinion, users are often not interested in the architecture and thus I wouldn't start with it. Did you have any other thoughts?

File: vertical-pod-autoscaler/README.md [hunk 3]
```diff
@@ -1,14 +1,46 @@
...
+Autoscaling is configured with a
+[Custom Resource Definition object](https://kubernetes.io/docs/concepts/api-extension/custom-resources/) 
+called [VerticalPodAutoscaler](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/apis/poc.autoscaling.k8s.io/v1alpha1/types.go).
+It contains:
+
+* Specification with label selector that points to pods that should be autoscaled.
+
+* Status through which the recommended cpu/memory requests are provided.
+
+VPA can operate in 3 modes:
```
Comment (tkulczynski) [inline vertical-pod-autoscaler/README.md]: Modes are already explained in the "Quick start" section - please deduplicate.

File: vertical-pod-autoscaler/README.md (hunk 1 above)
Comment (mwielgus) [reply vertical-pod-autoscaler/README.md]: Merged.

File: vertical-pod-autoscaler/README.md (hunk 3 above)
Comment (mwielgus) [reply vertical-pod-autoscaler/README.md]: Deduplicated.

File: vertical-pod-autoscaler/README.md (hunk 2 above)
Comment (mwielgus) [reply vertical-pod-autoscaler/README.md]: Removed for developers.

**Pull request merged**